MapboxNavigation/Resources/sv.lproj/Localizable.strings

```diff
@@ -1,4 +1,4 @@
-﻿﻿/* Format for speech string; 1 = way name; 2 = distance */
+﻿/* Format for speech string; 1 = way name; 2 = distance */
```

Comment:
If you used Xcode’s File inspector to convert the file from UTF-16 to UTF-8, it left a [BOM](https://en.wikipedia.org/wiki/Byte_order_mark) at the beginning of the file. Xcode’s strings file copying action, which converts the file from text plist format to binary plist format, only recognizes the BOM when the file is encoded in UTF-16. The workaround is either to cut and paste the file’s contents back into the file in Xcode or to use another text editor (or a tool like `iconv`).